docs/1.27/datamodel-and-migrations/datamodel-POSTGRES-knum.mdx

```diff
@@ -166,7 +166,7 @@ generate:
     output: ./prisma-schema
 ```
 
-Now you can run `prisma generate` and the Prisma CLI will stored the Prisma GraphQLS schema in `./prisma-schema/prisma.graphql`.
+Now you can run `prisma generate` and the Prisma CLI will store the Prisma GraphQLS schema in `./prisma-schema/prisma.graphql`.
```

Comment:
Hi,

Thanks for catching this.

I can also see more more typo here, can you also fix that as well 😅 
```suggestion
Now you can run `prisma generate` and the Prisma CLI will store the Prisma GraphQL schema in `./prisma-schema/prisma.graphql`.
```